.github/workflows/opensource.yml

```diff
@@ -0,0 +1,45 @@
+name: Open Source
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+  workflow_dispatch:
+
+
+jobs:
```

Comment:
You should probably add condition, so this job is only runs on open-condo-platform...

Seems useless to test in condo PR state of main of open-condo-platform